CONTRIBUTING.md

```diff
@@ -36,10 +50,19 @@ For other place, both English and Japanese are acceptable.
 
 ## philosophy of design / 設計の方針
 
+Our first purpose is "to help get the top in the programming contests".
+To make it possible, online-judge-tools automates "mistakable and cumbersome manual work".
+
+Also, we are focusing on "online-judge-tools does not cause the penalties".
+Since online-judge-tools using web scraping, sometimes behavior is unstable.
```

Comment:
```suggestion
Since online-judge-tools does web scraping, the behavior is unstable.
```

-   `sometimes` の位置が変
-   そもそもこの場合の「不安定」は「個々の実行」の性質ではなくどちらかと言えば「プログラムそのもの」の性質のはずで「ときどき不安定」というのは奇妙に聞こえる